src/system-health/scripts/healthd

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python2
+
+"""
+    healthd
+    System health monitor daemon for SONiC
+"""
+
+import signal
+import threading
+
+from sonic_py_common.daemon_base import DaemonBase
+from sonic_py_common.logger import Logger
+from swsssdk import SonicV2Connector
+
+from health_checker.manager import HealthCheckerManager
+
+SYSLOG_IDENTIFIER = 'healthd'
+logger = Logger(SYSLOG_IDENTIFIER)
```

Comment:
This logger is unnecessary. DaemonBase class now inherits from Logger, so HealthDaemon class should have built-in logging functionality. See this PR for examples: https://github.com/Azure/sonic-buildimage/pull/5370

Reply:
Fixed